config/lava/kselftest/kselftest.jinja2

```diff
@@ -27,3 +27,4 @@
         BOARD: {{ device_type }}
         BRANCH: {{ tree }}
         SKIP_INSTALL: True
+	ENV: '{{ env }}'
```

Comment:
I would drop this change, as this template is only used for the legacy system and therefore not relevant for your use-case. Otherwise LGTM.

Reply:
Thanks, I updated the commit